<?xml version="1.0"?>

<launch>
<arg name="ros_port" default="0"/>
<arg name="gaz_port" default="0"/>
<arg name="ros_ip" default="0"/>
<param name="use_sim_time" value="true"/>
<arg name="drone_name" default="iris"/>
  <group ns="$(arg drone_name)">
    <node name="landing_simulation_gym_node" pkg="training_q_learning" type="training.py" output="screen" >
        <param name = "drone_name" value="$(arg drone_name)"/>        
    </node> 
    <!-- <node name="copy_parameters" pkg="training_q_learning" type="copy_parameters.py" output="screen" >
    </node> 
    <node name="log_topic_freqs" pkg="training_q_learning" type="log_topic_freqs.py" output="screen" >
    </node>  -->
  </group>
</launch>